setup.sh

```diff
@@ -1,19 +1,50 @@
 #!/bin/bash
 
+# This script installs all the juju-1 and the dependencies.
+
+# Refresh the potentially stale apt cache.
 apt-get update -qq
+# Install software tools such as add-apt-repository.
 apt-get install -qy software-properties-common
+# Install the ppa for Juju and charm tools.
 apt-add-repository -y ppa:juju/stable
+# Refresh the apt-cache with the new Juju PPA.
 apt-get update -qq
 
-## don't install lxc & local till we get local support
-#apt-get install -qy lxc iptables juju-local
-
-## actually, we need juju-local since the local provider
-## can be used by bootstrapping from outside the container
-apt-get -qy install juju-local
-apt-get -qy install juju-core juju-quickstart juju-deployer python-jujuclient
```

Comment:
No more quickstart in jujubox?

The astute among you will notice I've mentioned this twice now.  Why are we dropping the only supported juju-1 deployer from jujubox?  Yes, I like juju-deployer.  Please add juju-quickstart too.